examples/tutorials/hnn_tutorial.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>Please add a line or two about the symplectic gradient and include references for figures and literature.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>